projects/igniteui-angular/src/lib/expansion-panel/expansion-panel.component.ts

```diff
@@ -97,7 +98,16 @@ export class IgxExpansionPanelComponent implements IgxExpansionPanelBase, AfterC
      * ```
      */
     @Input()
-    public collapsed = true;
+    public get collapsed(): boolean {
+        return this._collapsed;
+    }
+    public set collapsed(value) {
+        this._collapsed = value;
+        this.collapsedChange.emit(this._collapsed);
+    }
+
+    @Output()
+    public collapsedChange = new EventEmitter<any>();
```

Comment:
@MayaKirova There are 2 other events: onCollapsed/onExpanded that duplicate this event. Maybe we should remove them. What do you think?

Reply:
@mpavlinov That would be a breaking change. onCollapsed/onExpanded emit additional information - the event and panel that was collapsed/expanded, while the Change events emit just the current value. I don't think they can replace each other.